storage/client/blob/src/main/java/com/azure/storage/blob/implementation/AzureBlobStorageImpl.java

```diff
@@ -62,6 +61,20 @@ AzureBlobStorageImpl version(String version) {
         return this;
     }
 
+    /**
+     * The HTTP pipeline to send requests through.
+     */
+    private HttpPipeline httpPipeline;
+
+    /**
+     * Gets The HTTP pipeline to send requests through.
+     *
+     * @return the httpPipeline value.
+     */
+    public HttpPipeline httpPipeline() {
+        return this.httpPipeline;
+    }
```

Comment:
Even though this is not public API I'll ask anyway: is this required to be exposed? I would suggest removing it until it is required.

Reply:
This is used at https://github.com/Azure/azure-sdk-for-java/blob/593ebf2d056dd6096dd2a56307a8711a581c1c93/storage/client/blob/src/main/java/com/azure/storage/blob/implementation/AppendBlobsImpl.java#L61